src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -59,7 +97,23 @@ public function getDefinition()
     {
         return new FixerDefinition(
             'In method arguments and method call, there MUST NOT be a space before each comma and there MUST be one space after each comma.',
-            array(new CodeSample("<?php\nfunction sample(\$a=10,\$b=20,\$c=30) {}\nsample(1,  2);"))
+            array(
+                new CodeSample(
+                    "<?php\nfunction sample(\$a=10,\$b=20,\$c=30) {}\nsample(1,  2);",
+                    null
+                ),
+                new CodeSample(
+                    "<?php\nfunction sample(\$a=10,\$b=20,\$c=30) {}\nsample(1,  2);",
+                    array('keepMultipleSpacesAfterComma' => false)
+                ),
+                new CodeSample(
+                    "<?php\nfunction sample(\$a=10,\$b=20,\$c=30) {}\nsample(1,  2);",
+                    array('keepMultipleSpacesAfterComma' => true)
+                ),
+            ),
+            null,
+            'Configure to retain multiple spaces after comma.',
```

Comment:
how to configure it ? If one will type `php-cs-fixer describe method_argument_space` he is supposed to see all information how to configure it. Here, he just see that it is possible, but not how.